server/middleware/isAuthenticated.ts

```
@@ -0,0 +1,12 @@
+import { Request, Response, NextFunction } from 'express';
+import logger from '../logger';
+
+export function isAuthenticated(req: Request, res: Response, next: NextFunction) {
+  const requestId = (req as any).id || 'unknown';
+  if (req.isAuthenticated && req.isAuthenticated()) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)